### DISCUSSION TITLE
Is a "Tab Layout" possible with yabai?

### DISCUSSION THREAD

**Discussion by ispringle** [asmvik/yabai#1364] [Q&A] [answered]
i3's default fullscreen, or Xmonad's "Tabbed"  layout is what I'm looking to mimic. Essentially, a series of fullscreen (maybe in the macOS sense, full-sized) windows that are layed one on top of the other and then there are tabs at the top, a la browser tabs, that display the windows available and the active one. Honestly, I'd even be okay without the tabs if I could find a way to cycle through a number of full-sized windows.

My use case is that I have a workspace for my chat apps -- Slack, Teams, Discord, Signal -- and I want to keep them all in one workspace, however when they each take up 1/4th of the screen none are actually useable.

Another option might be that the active window takes up the majority of the screen and the inactive windows are tiled to the side, then cycling the active window shifts that window into the larger space, this is similar to how Xmonad and Bwspwm work.

**Comment by ispringle** [comment, marked as answer]
I found the answer as soon as I stopped trying to search for the term "tabbed". It appears I'm looking for the "stack" option... 🤦‍♂️

**Reply by es183923** [reply]
You can also get something similar to "tabs" by using https://github.com/AdamWagner/stackline.
